proof/refine/Invariants_H.thy

```diff
@@ -3115,48 +2642,16 @@ lemma sch_act_wf_cases:
     ResumeCurrentThread \<Rightarrow> ct_in_state' activatable'
   | ChooseNewThread     \<Rightarrow> \<top>
   | SwitchToThread t    \<Rightarrow> \<lambda>s. st_tcb_at' runnable' t s \<and> tcb_in_cur_domain' t s)"
-by (cases action) auto
-end
+  by (cases action) auto
 
-lemma (in PSpace_update_eq) cteCaps_of_update[iff]: "cteCaps_of (f s) = cteCaps_of s"
+lemma (in pspace_update_eq') cteCaps_of_update[iff]: "cteCaps_of (f s) = cteCaps_of s"
   by (simp add: cteCaps_of_def pspace)
 
 lemma vms_sch_act_update'[iff]:
   "valid_machine_state' (ksSchedulerAction_update f s) =
    valid_machine_state' s"
   by (simp add: valid_machine_state'_def )
 
-context begin interpretation Arch . (*FIXME: arch-split*)
-
-lemmas bit_simps' = pteBits_def asidHighBits_def asidPoolBits_def asid_low_bits_def
-                    asid_high_bits_def bit_simps
-
-lemma objBitsT_simps:
-  "objBitsT EndpointT = epSizeBits"
-  "objBitsT NotificationT = ntfnSizeBits"
-  "objBitsT CTET = cteSizeBits"
-  "objBitsT TCBT = tcbBlockSizeBits"
-  "objBitsT UserDataT = pageBits"
-  "objBitsT UserDataDeviceT = pageBits"
-  "objBitsT KernelDataT = pageBits"
-  "objBitsT (ArchT PTET) = word_size_bits"
-  "objBitsT (ArchT ASIDPoolT) = pageBits"
-  "objBitsT (ArchT VCPUT) = vcpuBits"
-  unfolding objBitsT_def makeObjectT_def
-  by (simp add: makeObject_simps objBits_simps bit_simps')+
-
-
-lemma objBitsT_koTypeOf :
-  "(objBitsT (koTypeOf ko)) = objBitsKO ko"
-  apply (cases ko; simp add: objBits_simps objBitsT_simps)
-  apply (rename_tac arch_kernel_object)
-  apply (case_tac arch_kernel_object; simp add: archObjSize_def objBitsT_simps bit_simps')
-  done
-
-lemma typ_at_aligned':
-  "\<lbrakk> typ_at' tp p s \<rbrakk> \<Longrightarrow> is_aligned p (objBitsT tp)"
```

Comment:
Is this no longer needed? It seems to have been removed completely.

Reply:
correct, it's not mentioned again in l4v/proofs